### PR TITLE
feat: include owned pets in login notifications

### PR DESCRIPTION
**Pull request by iProdigy**
Relies upon base RL chat commands plugin

### CODE REVIEW

File: src/main/java/dinkplugin/notifiers/data/LoginNotificationData.java [hunk 1]
```diff
@@ -21,6 +21,8 @@ public class LoginNotificationData extends NotificationData {
     Progress questCount;
     Progress questPoints;
     SlayerData slayer;
+    @Nullable // requires Chat Commands plugin to be enabled
+    Map<Integer, String> petNamesById;
```
Comment (iProdigy) [inline src/main/java/dinkplugin/notifiers/data/LoginNotificationData.java:25]:
```suggestion
    Map<Integer, String> pets;
```

Alternatively we can use a simpler name & explain in readme

File: README.md [hunk 2]
```diff
@@ -834,13 +834,19 @@ On login, Dink can submit a character summary containing data that spans multipl
     "slayer": {
       "points": 2204,
       "streak": 1074
+    },
+    "pets": {
+      "11995": "Pet chaos elemental",
+      "13071": "Chompy chick"
```
Comment (pajlada) [inline README.md:840]: Is the key not an integer?

Comment (iProdigy) [reply README.md:840]: JSON spec unfortunately forces them into strings

**Review by Felanbird — APPROVED** [review]
```
      "pets":[
         {
            "itemId":22746,
            "name":"Ikkle hydra"
         },
         {
            "itemId":11995,
            "name":"Pet chaos elemental"
         },
         {
            "itemId":12646,
            "name":"Baby mole"
         },
         {
            "itemId":12655,
            "name":"Pet kraken"
         },
         {
            "itemId":21273,
            "name":"Skotos"
         },
         {
            "itemId":12648,
            "name":"Pet smoke devil"
         },
         {
            "itemId":13320,
            "name":"Heron"
         },
         {
            "itemId":26901,
            "name":"Abyssal protector"
         }
      ]
   },
```

**Review by pajlada — APPROVED** [review]

**Pull request merged**